### PR TITLE
Feature/post tag

### PR DESCRIPTION
**Pull request by seung-00**
## Summary
* add category and date to the bottom of Card

### CODE REVIEW

**Comment by dddieon** [comment]
LG
TM

**Review by dddieon — APPROVED** [review]

File: src/styles/components/common/card.scss [hunk 1]
```diff
@@ -115,10 +110,12 @@
         }
         &-text {
           @include point-font();
+          color: #b8b8b8;
```
Comment (jeongnaehyeok) [inline src/styles/components/common/card.scss]:
```suggestion
          color: $light-gray;
```

File: src/styles/components/common/card.scss [hunk 2]
```diff
@@ -132,6 +129,8 @@
...
```
Comment (jeongnaehyeok) [inline src/styles/components/common/card.scss]:
```suggestion
          color: $light-gray;
```

File: src/styles/components/common/card.scss (hunk 1 above)
Comment (jeongnaehyeok) [reply src/styles/components/common/card.scss]: 색 변수로 넣어주세요~~

**Review by jeongnaehyeok — APPROVED** [review]
LGTM👍

**Pull request merged**